### PR TITLE
Update rails & ruby

### LINKED ISSUE #157: Upgrade to ruby 3.0.x and rails 6.0.x

### PR DESCRIPTION
**Pull request by bsp3ars**
Resolves #157 

Here is the current status of the branch:
* Ruby and Rails have been updated to 3.0.4p208 and 6.0.6 respectively
* The code works in development mode
* One feature test is kinda wonky, specifically I couldn't get it to wait long enough to find an id after a page change (spec/features/create_admin_spec.rb lines 36-38)
* The capistrano deploy to QA is failing due to `ArgumentError: Missing secret_key_base for 'production' environment, set this string with rails credentials:edit`

Recent Troubleshooting Steps:
1. I added the secret_key_base to the .env.production.local on QA but it didn't resolve the issue
2. Started to look into the Rails 6 changes regarding how secret_key_base is stored and referenced

Other things of note:
* You can unset the bundle installation location (vendor/bundle) if you accidentally set it via: `bundle config unset path`
* I used the following sites for reference when upgrading Ruby and Rails (https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) (https://selleo.com/blog/how-to-upgrade-to-rails-6)
* You can use the d option to show the difference when using the bundle update rails command

### CODE REVIEW

**Comment by crowesn** [comment]
* Corrected .env filename in Capistrano config
* add rbenv config to :production deploy config
* Created new secret key

**Comment by crowesn** [comment]
@lisa3711 @scherztc 
this should be ready to go.

File: spec/features/create_admin_spec.rb [hunk 1]
```diff
@@ -27,13 +27,15 @@
...
-    fill_in('password', with: ENV['ADMIN_PASSWORD'])
+    fill_in('username', with: ENV.fetch('ADMIN_USERNAME', nil))
+    fill_in('password', with: ENV.fetch('ADMIN_PASSWORD', nil))
     click_on('Submit')
-
     expect(page).to have_current_path(Rails.application.routes.url_helpers.publications_path)
 
     # Check values
+    page.current_window.resize_to(1920, 1080)
+    page.save_screenshot('~/screenshot.png')
```
Comment (scherztc) [inline spec/features/create_admin_spec.rb:37]: Why did we need this?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -108,7 +108,7 @@
   #   # order dependency and want to debug it, you can fix the order by providing
   #   # the seed, which is printed after each run.
   #   #     --seed 1234
-  #   config.order = :random
+  config.order = :random
```
Comment (scherztc) [inline spec/spec_helper.rb:111]: Why did we need this?

**Comment by scherztc** [comment]
Coveralls : Coverage report generated for RSpec to /Users/scherztc/Workspaces/aaec/coverage. 641 / 697 LOC (91.97%) covered.

**Comment by scherztc** [comment]
The main thing that wasn't well documented and therefore unable to test:

1.  How do I login as the admin user to see all of the publications?

File: .circleci/config.yml [hunk 3]
```diff
@@ -62,28 +81,34 @@ jobs:
           command: |
             gem install rubocop
             bundle exec rubocop
+
+      # Brakeman
```
Comment (scherztc) [inline .circleci/config.yml:85]: Do we want these?

**Comment by scherztc** [comment]
We ran the test suite locally, read through the code and made comments, and spun up the application, reviewed circleci, and clicked through the available features.

We still need to do a test deploy with Capistrano.

**Review by scherztc — COMMENTED** [review]
We ran the test suite locally, read through the code and made comments, and spun up the application, reviewed circleci, and clicked through the available features.

We still need to do a test deploy with Capistrano.

**Pull request merged**